images/nfs/start.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
This file needs a license also, what was the license on the upstream version?  we should give attribution here, similar to what we did for the Dockerfile

Reply:
I am using  parts from this https://github.com/mitcdh/docker-nfs-ganesha/blob/master/start.sh and it doesn't have license